nixos/modules/services/networking/openvpn.nix

```
@@ -181,6 +191,12 @@ in
             '';
           };
 
+          authUserPassFile = mkOption {
+            default = null;
+            description = lib.mdDoc ''Specify a file path to use as `auth-user-pass ''${authUserPass}`. File will not be included in the nix store unless you specify it as a path `authUserFile = ./myauthfile`'';
+            type = types.nullOr types.str;
```

We can use `types.path` as that will validate that the value is a path plausibly.

Until something like https://github.com/NixOS/nixpkgs/pull/78640 is merged, we don't really have a better option.

```suggestion
            type = types.nullOr types.path;
```

---

Doesn't that mean that it will be a path of the type that will be included in the store? I specifically mean to avoid that, because it will contain credentials.